vmdb/app/controllers/application_controller.rb

```diff
@@ -2713,7 +2713,7 @@ def list_row_id(row)
     to_cid(row['id'])
   end
 
-  def list_row_image(image_path, image, model_image, _itemname)
+  def list_row_image(image_path, image, model_image, _item)
     image ? image : "#{image_path}#{model_image}.png"
```

Comment:
@chessbyte Made the recommended suggestion.
This is definitely great compared to the ternary operator used previously.